test/forms/allowances/CapitalAllowancesForACarFormProviderSpec.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.allowances
+
+import forms.allowances.CapitalAllowancesForACarFormProvider
+import forms.behaviours.BooleanFieldBehaviours
+import models.CapitalAllowancesForACar
+import org.scalatest.OptionValues
+import play.api.data.FormError
+
+class CapitalAllowancesForACarFormProviderSpec extends BooleanFieldBehaviours with OptionValues {
```

Comment:
It seems that we are not utilizing OptionValues. Is there a need for them?

Reply:
Looks like I need "BooleanFieldBehviours" because it's a YesNo page? But you are right with "OptionValues" so I've removed it